File: majorProject/src/main/java/com/aocc/framework/NoOpMusic.java
package com.aocc.framework;

public class NoOpMusic implements Music {

    private float volume;
    private boolean looping;

    @Override
    public void play() {
    }

    @Override
    public void stop() {
    }

    @Override
    public void pause() {
    }

    @Override
    public void setLooping(boolean looping) {
        this.looping = looping;
    }

    @Override
    public void setVolume(float volume) {
        this.volume = volume;
    }

    @Override
    public boolean isPlaying() {
        return false;
    }

    @Override
    public boolean isStopped() {
        return true;
    }

    @Override
    public boolean isLooping() {
        return looping;
    }

    @Override
    public void dispose() {
    }

    @Override
    public void seekBegin() {
    }
}

File: majorProject/src/main/kotlin/com/aocc/framework/NoOpMusic.kt
package com.aocc.framework

class NoOpMusic : Music {

    private var volume: Float = 0f
    private var looping: Boolean = false

    override fun play() {
    }

    override fun stop() {
    }

    override fun pause() {
    }

    override fun setLooping(looping: Boolean) {
        this.looping = looping
    }

    override fun setVolume(volume: Float) {
        this.volume = volume
    }

    override fun isPlaying(): Boolean {
        return false
    }

    override fun isStopped(): Boolean {
        return true
    }

    override fun isLooping(): Boolean {
        return looping
    }

    override fun dispose() {
    }

    override fun seekBegin() {
    }
}
